astroquery/dastcom5/tests/test_dastcom5.py

```diff
@@ -0,0 +1,102 @@
+import os
+
+import numpy as np
+import pytest
+from pytest_mock import mocker
+
+from ...dastcom5 import Dastcom5
+
+from astropy.table import Table
+
+
+def test_asteroid_db_is_called_with_right_path(mocker):
+    mock_np_fromfile = mocker.patch("poliastro.neos.dastcom5.np.fromfile")
+    mock_open = mocker.patch("poliastro.neos.dastcom5.open")
+    Dastcom5.asteroid_db()
+    mock_open.assert_called_with(Dastcom5.ast_path, "rb")
+
+
+def test_comet_db_is_called_with_right_path(mocker):
+    mock_np_fromfile = mocker.patch("poliastro.neos.dastcom5.np.fromfile")
+    mock_open = mocker.patch("poliastro.neos.dastcom5.open")
+    Dastcom5.comet_db()
+    mock_open.assert_called_with(Dastcom5.com_path, "rb")
+
+
+def test_read_headers(mocker):
+    mock_np_fromfile = mocker.patch("poliastro.neos.dastcom5.np.fromfile")
+    mock_open = mocker.patch("poliastro.neos.dastcom5.open")
+    Dastcom5.read_headers()
+    mock_open.assert_any_call(
+        os.path.join(Dastcom5.dbs_path, "dast5_le.dat"), "rb"
+    )
+    mock_open.assert_any_call(
+        os.path.join(Dastcom5.dbs_path, "dcom5_le.dat"), "rb"
+    )
+
+
+def test_read_record(mocker):
+    mock_np_fromfile = mocker.patch("poliastro.neos.dastcom5.np.fromfile")
```

Comment:
why do you need to mock this?  `np.fromfile` should be directly usable?

Reply:
I wanted to assert if that function is being called. For that I have used the method provided by mock.
Like this: https://github.com/astropy/astroquery/pull/1339/files#diff-b7d60e112ed25e166e296e2824e9f2a7R75